assets/css/style.css

```diff
@@ -5,25 +5,40 @@ h1 {
 .wrapper {
   margin-top: 5rem;
 }
+
 .wrapper .row {
   width: 100%;
   margin: 0 auto;
+  display: -webkit-box;
+  display: -ms-flexbox;
   display: flex;
-  justify-content: center;
-  perspective: 1000px;
+  -webkit-box-pack: center;
+      -ms-flex-pack: center;
```

Comment:
indent is wrong... styles should all start at the same place

Reply:
it got automaticlly generated by my sass compiler